homeassistant/components/media_player/browse_media.py

```diff
@@ -159,3 +163,29 @@ def calculate_children_class(self) -> None:
     def __repr__(self) -> str:
         """Return representation of browse media."""
         return f"<BrowseMedia {self.title} ({self.media_class})>"
+
+
+@dataclass(kw_only=True, frozen=True)
+class SearchMedia:
+    """Represent a search media file."""
```

Comment:
It's not really a file, but a search result container. 
```suggestion
    """Represent search results."""
```